common/thorhelper/thorread.cpp

```diff
@@ -14,6 +14,9 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 ############################################################################## */
+#ifdef _USE_PARQUET
```

Comment:
trivial: Probably clearer, and less likely to have include problems, if it comes after all the other #includes.

Reply:
I tried moving the #include to the bottom, but that caused include problems in the platform code and apache arrow code. I tried shuffling a few other #includes around in the parquetembed files without any success. How should I approach this in order to find the problem #include statement?